Entities/Timeslot.cs

```diff
@@ -0,0 +1,13 @@
+using FRITeam.Swapify.Entities.Enums;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace FRITeam.Swapify.Entities
+{
+    public class TimeSlot : BaseEntity
```

Comment:
maybe struct is enough (it is like a Point), and is not descendant of BaseEntity

Reply:
OK